### DISCUSSION TITLE
Is it possible to exclude specific queries like `grep -v` option?

### DISCUSSION THREAD

**Discussion by noraworld** [BurntSushi/ripgrep#2183] [Q&A] [answered]
I would like to do something like this:

```shell
rg word_i_want_to_search_for -v word_i_do_not_want_to_search_for
```

Is it possible?

**Comment by BurntSushi** [comment, marked as answer]
Well I mean, `grep` doesn't do that either. Instead, you'd want `grep what-i-want some-file | grep -v what-i-dont-want`. It's the same for ripgrep! For a recursive search of the current directory, just do `rg what-i-want ./ | rg -v what-i-dont-want`.

Outside of some corner cases, in general, if something works for `grep`, then it will also work for `rg`. So you should _just try it first_. The main issue here is that the command you've posted _doesn't actually work for `grep`._

See also: #875

**Reply by noraworld** [reply]
Oh I'm sorry, I completely thought `grep` could do that, and I didn't confirm it.

Thank you for letting me know. I probably understood what I really want to do.
